### PR TITLE
LTC 2 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/galadriel_elven_queen.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Galadriel, Elven-Queen
+ManaCost:2 G U
+Types:Legendary Creature Elf Noble
+PT:4/5
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | IsPresent$ Elf.Other+ThisTurnEntered+EnteredUnder You | TriggerZones$ Battlefield | Execute$ TrigVote | TriggerDescription$ Will of the council — At the beginning of combat on your turn, if another Elf entered the battlefield under your control this turn, starting with you, each player votes for dominion or guidance. If dominion gets more votes, the Ring tempts you, then you put a +1/+1 counter on your Ring-bearer. If guidance gets more votes or the vote is tied, draw a card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/galadriel_elven_queen.txt:5]: wrong condition

File: forge-gui/res/cardsfolder/upcoming/travel_through_caradhras.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Travel Through Caradhras
+ManaCost:5 G
+Types:Sorcery
+A:SP$ Vote | Defined$ Player | StoreVoteNum$ True | VoteType$ RedhornPass,MinesofMoria | SubAbility$ VoteRedhornPass | SpellDescription$ Council's dilemma — Starting with you, each player votes for Redhorn Pass or Mines of Moria. For each Redhorn Pass vote, search your library for a basic land card and put it onto the battlefield tapped. If you search your library this way, shuffle. For each Mines of Moria vote, return a card from your graveyard to your hand.
+SVar:VoteRedhornPass:DB$ ChangeZone | Origin$ Library | Destination$ Battlefield | ChangeType$ Land.Basic | ChangeNum$ VoteNumRedhornPass | Tapped$ True | Mandatory$ True | SubAbility$ VoteMinesofMoria
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/travel_through_caradhras.txt:5]: Mandatory wrong

File: forge-gui/res/cardsfolder/upcoming/travel_through_caradhras.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+SVar:VoteMinesofMoria:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | Mandatory$ True | ChangeType$ Permanent.YouOwn | SelectPrompt$ Return a card from your graveyard to your hand for each Mines of Moria vote | Hidden$ True | ChangeNum$ VoteNumMinesofMoria | SubAbility$ DBChange
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/travel_through_caradhras.txt:6]: wrong type

File: forge-gui/res/cardsfolder/upcoming/galadriel_elven_queen.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+ManaCost:2 G U
+Types:Legendary Creature Elf Noble
+PT:4/5
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | CheckSVar$ ElfEntered | TriggerZones$ Battlefield | Execute$ TrigVote | TriggerDescription$ Will of the council — At the beginning of combat on your turn, if another Elf entered the battlefield under your control this turn, starting with you, each player votes for dominion or guidance. If dominion gets more votes, the Ring tempts you, then you put a +1/+1 counter on your Ring-bearer. If guidance gets more votes or the vote is tied, draw a card.
+SVar:TrigVote:DB$ Vote | Defined$ Player | VoteType$ Dominion,Guidance | VoteDominion$ DBTempt | VoteGuidance$ DBDraw | Tied$ DBDraw
+SVar:DBTempt:DB$ RingTemptsYou | SubAbility$ DBPutCounter
+SVar:DBPutCounter:DB$ PutCounter | Choices$ Card.IsRingbearer+YouCtrl | Mandatory$ True | CounterType$ P1P1 | CounterNum$ 1 
+SVar:DBDraw:DB$ Draw
+SVar:ElfEntered:Count$ThisTurnEntered_Battlefield_Creature.Elf+StrictlyOther+YouCtrl
+DeckHints:Type$Elf
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/galadriel_elven_queen.txt:11]: maybe Needs? At least at first glance, she doesn't do anything without other Elves

File: forge-gui/res/cardsfolder/upcoming/travel_through_caradhras.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Travel Through Caradhras
+ManaCost:5 G
+Types:Sorcery
+A:SP$ Vote | Defined$ Player | StoreVoteNum$ True | VoteType$ RedhornPass,MinesofMoria | SubAbility$ VoteRedhornPass | SpellDescription$ Council's dilemma — Starting with you, each player votes for Redhorn Pass or Mines of Moria. For each Redhorn Pass vote, search your library for a basic land card and put it onto the battlefield tapped. If you search your library this way, shuffle. For each Mines of Moria vote, return a card from your graveyard to your hand.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/travel_through_caradhras.txt:4]: for UI could you try `VoteType$ Redhorn Pass,Mines of Moria`
You will need to similarly space VoteNumRedhorn Pass
But I think it should work?
No spaces needed in SubAbility names!

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/travel_through_caradhras.txt:4]: huh this acually works. Thanks

File: forge-gui/res/cardsfolder/upcoming/travel_through_caradhras.txt (hunk 2 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/travel_through_caradhras.txt]: you removed the wrong one

**Review by tool4ever — APPROVED** [review]

**Pull request merged**